### ISSUE TITLE
OS X Support

### ISSUE THREAD

**Issue by airhorns** [facebookarchive/presto-odbc#52]
It would be amazing to use Presto as a datasource from OS X. Tableau has a stable OS X desktop version now, and this would be great. 

How much work would be required for OS X support?

**Comment by markisaa** [comment]
@airhorns - At the moment this project has no full time maintainer. There are still plenty of kinks to work out on the Windows platform to make this project more widely usable. Making it work on OSX is as simple (or hard) as getting this to work: https://github.com/prestodb/presto-odbc/blob/master/driver/driver.d#L61

Modulo a few other small items. When I developed the driver last year, that function didn't seem to want to execute when I needed it to. There is very limited support for D language shared objects on OSX as I understand it; perhaps that has changed given that there have been some new compiler releases since then.

**Comment by electrum** [comment]
Related to ODBC in general, Teradata has committed to supporting Presto including ODBC: http://www.teradata.com/Presto/

However, I don't know of specifics aside from the _"These enhancements will be
completed and available in 2016."_ statement on their datasheet. @mattsfuller any additional details here?

**Comment by mattsfuller** [comment]
Yes, the plan is for the ODBC driver to be a robust and complete per the spec. And will be supported on multiple platforms. It will be available sometime before the 2nd half of 2016. Hopefully sooner.
